widget/contrib/updates.lua

```diff
@@ -0,0 +1,84 @@
+--[[
+
+     Licensed under GNU General Public License v2
+      * (c) 2017, Alphonse Mariyagnanaseelan
+
+--]]
+
+local helpers       = require("lain.helpers")
+local awful         = require("awful")
+local shell         = require("awful.util").shell
+local wibox         = require("wibox")
+local naughty       = require("naughty")
+local lines, floor  = io.lines, math.floor
+local string        = { format = string.format,
+                        gsub   = string.gsub,
+                        len    = string.len}
+
+-- Available updates
+-- lain.widget.contrib.updates
+
+local function factory(args)
+    local updates          = { widget = wibox.widget.textbox() }
+    local args            = args or {}
+    local timeout         = args.timeout or 900
+
+    local notify          = args.notify or false
+    local notify_title    = args.notify_title or "Updates"
+    local notify_timeout  = args.notify_timeout or 15
+
+    local settings        = args.settings or function() end
+    local package_manager = args.package_manager
+    local command         = args.command or ""
+
+    local commands = {
+        pacman        = "checkupdates | sed 's/->/→/' | column -t",
```

Comment:
What is this `checkupdates`? Remember not to add custom scripts calls or new dependencies.

Reply:
`checkupdates` is a script provided by `pacman`, the Arch package manager.